src/app/components/Home/Office/Canvas/PopUpInfoPlace/index.tsx

```diff
@@ -176,14 +180,21 @@ const PopUpInfoPlace: React.FC<IPopUpInfoPlace> = ({
         ...prevState,
         occupant: user,
         startReservation: event.target.value,
+        endReservation: event.target.value,
       }));
     }
   };
 
-  const handleEventButton = () => {
+  const handleEventButtonSave = () => {
     dispatch(setReservation(dataSelectedWorkplace));
   };
 
+  const handleEventButtonCancel = () => {
+    const dataSelectedWorkplaceAfterClearing = clearBooking(dataSelectedWorkplace)
+    setDataSelectedWorkplace(dataSelectedWorkplaceAfterClearing);
+    dispatch(setReservation(clearBooking(dataSelectedWorkplace)));
```

Comment:
`dataSelectedWorkplaceAfterClearing` instead of `clearBooking(dataSelectedWorkplace)`